hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/update/server/MockCRLStore.java

```diff
@@ -107,7 +107,7 @@ public BigInteger issueCert() throws Exception {
         scmCertStore.getCrls(ImmutableList.of(crlId.get()));
 
     if (crlInfos.isEmpty()) {
-      log.debug("CRL[0]: {}", crlInfos.get(0).toString());
+      log.debug("CRL[0]: {}", crlInfos.get(0));
```

Comment:
this statement doesn't make sense.
if crlInfos is empty, crlInfos.get(0) would throw IndexOutOfBoundsException

Reply:
@xiaoyuyao any idea what this was meant for?